Assets/Tests/InputSystem/SwitchTests.cs

```diff
@@ -80,6 +80,61 @@ private static SwitchProControllerHIDInputState StateWithButton(SwitchProControl
         }.WithButton(button);
     }
 
+    [Test]
+    [Category("Devices")]
+    public void Devices_SwitchProAxisJitter_DoesntMakeDeviceCurrent()
+    {
+        var device1 = InputSystem.AddDevice<SwitchProControllerHID>();
+        var device2 = InputSystem.AddDevice<SwitchProControllerHID>();
+
+        InputSystem.QueueStateEvent(device1,
+            new SwitchProControllerHIDInputState
+            {
+                leftStickX = 0x7f,
+                leftStickY = 0x7f,
+                rightStickX = 0x7f,
+                rightStickY = 0x7f,
+            });
+        InputSystem.QueueStateEvent(device2,
+            new SwitchProControllerHIDInputState
+            {
+                leftStickX = 0x7f,
+                leftStickY = 0x7f,
+                rightStickX = 0x7f,
+                rightStickY = 0x7f,
+            });
+        InputSystem.Update();
+        Assert.That(Gamepad.current, Is.EqualTo(device2));
+
+        var jitterMask = 0b111;
+
+        InputSystem.QueueStateEvent(device1,
+            new SwitchProControllerHIDInputState
+            {
+                leftStickX = (byte)(0x7f - jitterMask),
```

Comment:
I don't think I understand what's going on here. Why device1 didn't become current after this?
The result is 0b0111'1000, so there are 4 set bits outside of the jitter range that are still set and should still be picked up as a change? Or am I missing something?